cloudify/workflows/local.py

```diff
@@ -505,9 +514,11 @@ def __init__(self, storage_dir='/tmp/cloudify-workflows'):
         self._data_path = None
         self._payload_path = None
         self._blueprint_path = None
+        self.created_at = None
```

Comment:
It gets already initialized in the parent class, no?

Reply:
this one doesn't call super (for some reason)

Reply:
Isn't it doing that on line 509 (of the modified version)?

Reply:
oh yeah it does now, lol